cmdeploy/src/cmdeploy/service/filtermail-incoming.service.f

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Incoming Chatmail Postfix before queue filter 
+
+[Service]
+ExecStart={execpath} {config_path} incoming
```

Comment:
this could have been a single service, but I guess it is interesting to see which one causes more memory usage, OTOH what is likely to happen is that both will use more memory usage than having a single service